contrib/linux/actions/service.py

```diff
@@ -20,20 +20,25 @@
 import platform
 import subprocess
 
+from st2common.util.shell import quote_unix
+
 distro = platform.linux_distribution()[0]
 
-args = {'act': sys.argv[1], 'service': sys.argv[2]}
+if len(sys.argv) < 3:
+    raise ValueError('Usage: service.py <action> <service>')
+
+args = {'act': quote_unix(sys.argv[1]), 'service': quote_unix(sys.argv[2])}
```

Comment:
This is now not strictly needed since ``subprocess.Popen`` already takes case of that when a list of args and ``shell=False`` is used.